### ISSUE TITLE
@deck.gl/widgets v8.10.0-alpha.4 use React.createElement instead of the preact jsxFactory h

### ISSUE THREAD

**Issue by amarant** [visgl/deck.gl#8182]
### Description

Hi,
I was trying the @deck.gl/widgets v8.10.0-alpha.4 and it crashes because the translation from JSX to JS use `React.createElement` instead of the preact version `h`. I don't know why it didn't compile with it because the tsconfig seems to indicate the preact jsxFactory.

notify @chrisgervang who made the PR #8024 

### Flavors

- [ ] Script tag
- [X] React
- [ ] Python/Jupyter notebook
- [ ] MapboxOverlay
- [ ] GoogleMapsOverlay
- [ ] CartoLayer
- [ ] ArcGIS

### Steps to Reproduce

create a simple deck map using :

```
<DeckGL  widgets={[new FullscreenWidget({ id: 'test' })]} />
```

### Environment

- Framework version: v8.10.0-alpha.4
- Browser: Chrome
- OS:

### Logs

```
deck: React is not defined undefined
```

**Comment by chrisgervang** [comment]
How do you install deck.gl in your environment? I'm not expecting our npm bundles to include JSX. Are you building from source?

**Comment by amarant** [comment]
I use the npm bundle, and the esm or es5 distributions don't use jsx, but it uses `React.createElement` instead of the preact function `h`, just look at [Npm in @deck.gl/widgets/dist/esm/fullscreen-widget.js ](https://www.npmjs.com/package/@deck.gl/widgets/v/8.10.0-alpha.4?activeTab=code).

![Screenshot_20231008-110034](https://github.com/visgl/deck.gl/assets/2281471/5ef90096-91e9-4d50-b7fb-b4282794fdd9)

**Comment by chrisgervang** [comment]
Just published `8.10.0-alpha.5` with a fix. Please give it a try.
<img width="430" alt="Screen Shot 2023-10-17 at 4 08 36 PM" src="https://github.com/visgl/deck.gl/assets/2461547/d5e4544e-776d-4456-9ca8-b6747456af0b">

**Comment by amarant** [comment]
Thanks, this part works now, but there is an [error in the stylesheet](https://github.com/visgl/deck.gl/blob/69743353d537ce9e5b797e6cb18d597f1ca159cb/modules/widgets/src/stylesheet.css#L72).

**Comment by chrisgervang** [comment]
Released 8.10.0-alpha.6 with that fixed.

**Comment by amarant** [comment]
This is all fixed now, thanks.
